static/templates/right_sidebar.hbs

```diff
@@ -27,7 +27,8 @@
             <a id="invite-user-link" href="#invite"><i class="fa fa-user-plus" aria-hidden="true"></i>{{t 'Invite more users' }}</a>
             {{/if}}
             <a id="sidebar-keyboard-shortcuts" data-overlay-trigger="keyboard-shortcuts" class="hidden-for-spectators">
-                <i class="fa fa-keyboard-o fa-2x tippy-zulip-tooltip" id="keyboard-icon" data-tippy-allowHTML="true" data-tippy-content="{{t 'Keyboard shortcuts' }} &lt;span class='hotkey-hint'&gt;(?)&lt;/span&gt;"></i>
+                <i class="fa fa-keyboard-o fa-2x tippy-zulip-tooltip" id="keyboard-icon" data-tooltip-id="keyboard-icon-tooltip"></i>
```

Comment:
Would it be better to call it `data-tooltip-template-id`?  I feel like spelling that out may help with readability that this feature always will have an associated `<template>` element.

Reply:
It’s not like someone is going to treat the name of this attribute as complete documentation for this feature and be able to use it without referencing other examples.

I’m weakly negative on the extra verbosity, but I’ll change it to avoid unimportant squabbling.